index.html

```diff
@@ -264,350 +274,353 @@ <h2>Intro Section of the Document</h2>
-              <li>MUST contain information about the scenario that was tested, e.g. controlling the room temperature by measuring temperature and adjusting the heater.</li>
-              <li>MUST explain where discussions on implementation experience should be collected</li>
-              <li>SHOULD provide the history of all the past testing events (or explain how to retrieve the history of the results gathered during those events)</li>
-              <li>SHOULD contain a reference to the implementations of Consumers or Exposers.</li>
-              </ul>
-              </li>
-              <li>For the binding to transition to the &quot;Current&quot; state, a <em>Test Report</em> MUST exist. The <em>Test Report</em> MUST contain at least one implementation of a Consumer (capable of understanding and performing all the operations described in the binding) and one Exposer (capable of handling all the operations and features described in the binding and optionally be able to create a valid TD). Additional implementations can be added even after the transition to the Current.</li>
-              <li>The exact contents of the Test Report is not decided yet. See <a href="https://github.com/w3c/wot-bindings-registry/issues/3">https://github.com/w3c/wot-bindings-registry/issues/3</a></li>
-              <li>Submitters MAY call for transition but the custodian can also automatically trigger the process once it is verified that the condition above is reached.</li>
-              <li><em>Test Reports</em> and related resources SHOULD be published in a git repository. The repository SHOULD be public and it MUST be accessible to the reviewers and the custodian.</li>
-              <li>Collaboration between the custodian, reviewers, and submitters is highly encouraged, ideally through a Plugfest or another structured testing session where different implementations can be evaluated collectively.</li>
+      <section id="submission-requirements">
+        <h2>Submission Requirements</h2><!-- Requirements on the Submitted Document -->
+        
+        <p>What does the binding have to contain to go into the table.</p>
+        <ul>
+          <li><a href="#req-opmap"></a></li>
+          <li><a href="#req-contmap"></a></li>
+          <li><a href="#req-traninit"></a></li>
+          <li><a href="#req-changelog"></a>
+          <li><a href="#req-docsec"></a>
+          <li><a href="#req-copy"></a>
+          <li><a href="#req-openread"></a>
+          <li><a href="#req-openrev"></a>
+          <li><a href="#req-summ"></a>
+          <li><a href="#req-trancurr"></a>
+          <li><a href="#req-content"></a>
+          <li><a href="#req-docs"></a>
+          <li><a href="#req-confl"></a>
+          <li><a href="#req-redef"></a>
+          <li><a href="#req-deps"></a>
+          </li>
+        </ul>
+        
+        <p class="ednote">[DP] Should we create a table instead of the (sub-)sections?</p>
```

Comment:
Some are quite long and need explanations so I don't think they can really fit in a table